framework/src/geomsearch/FindContactPoint.C

```diff
@@ -49,8 +52,13 @@ findContactPoint(PenetrationInfo & p_info,
                  const Point & secondary_point,
                  bool start_with_centroid,
                  const Real tangential_tolerance,
-                 bool & contact_point_on_side)
+                 bool & contact_point_on_side,
+                 bool & search_succeeded,
+                 bool verbose)
 {
+  // Default to true and we'll switch on failures
+  search_succeeded = true;
```

Comment:
AFAICT you never switch this to `false`?

Reply:
added now